File: CodeChef/JulyChallenge2020/AdaKing/ADAKING.py
# cook your dish here
import sys 

#   below 2 lines are for running code on your local machine 
#   Remove these 2 lines while using code online on codechef as they provide thier own input
sys.stdin = open("input.txt",'r')
sys.stdout = open("output.txt",'w')


t = int(input())

for i in range(t):
    k = int(input())
    arr = list()
    for j in range(64):
        arr.append('X')
    
    arr[0]='O'
    for j in range(1,k):
        arr[j]='.'
    
    for j in range(64):
        print(arr[j],end='')
        if (j+1)%8==0:
            print(' ')
        

File: CodeChef/JulyChallenge2020/AdaKing/ADAKING.cpp
#include <iostream>
using namespace std;

int main() {
	// your code goes here

    //below 2 lines are for running code on your local machine 
    //Remove these 2 lines while using code online on codechef as they provide thier own input
    freopen("input.txt", "r", stdin);
    freopen("output.txt", "w", stdout);
    
	int t;
	cin>>t;
	for(int i=0;i<t;i++)
	{
	    int k;
	    cin>>k;
	    char arr[64];
	    for(int j=0;j<64;j++)
	    {
	        arr[j]='X';
	    }
	    arr[0]='O';
	    k--;
	    for(int j=1;j<=k;j++)
	    {
	        arr[j]='.';
	    }

	    for(int j=0;j<64;j++)
	    {
	       cout<<arr[j];
	       if((j+1)%8 ==0)
	        cout<<endl;
	    }
	}
	return 0;
}
